### PR TITLE
Multiple commit PR

### PR DESCRIPTION
**Pull request by flleeppyy**
- Added development script with NODE_ENV (win-node-env for Windows users) which will launch dev tools for you for most windows when active.
- Added certain yarn files to gitignore (error and log files)
- Updated bug_report to include comment telling user to use commit hash for version if they're using a build from the development branch
- Implement Lyrics always on top #746

### CODE REVIEW

**Comment by flleeppyy** [comment]
hold on Im trying to remember why I commented out a bunch of `'browserWindow' = null` lines. I think I was trying to figure out a bug where it would yell at you saying "Object is destroyed" whenever you closed the main window.

**Comment by flleeppyy** [comment]
fixed it...
_sigh_

File: main.js [hunk 1]
```diff
@@ -768,6 +773,7 @@ async function createWindow() {
                         infoPlayerProvider.getPlayerInfo().likeStatus !==
                         'DISLIKE'
                     ) {
+                        1257
```
Comment (mingjun97) [inline main.js:776]: What's this `1257` for? 🤣

**Review by mingjun97 — COMMENTED** [review]
Great job! Overall great, except for the magic number. What's it for? 🤣

File: main.js (hunk 1 above)
Comment (flleeppyy) [reply main.js:776]: I do not remember adding that actually, uhhh
it was probably me trying to type something in another window but forgot vscode was still in focus, and didn't realize i put that there

Comment (flleeppyy) [reply main.js]: Just fixed it

Comment (mingjun97) [reply main.js]: Cool! It happens sometime for me as well. Lol

File: STORE_VARIABLES.md [hunk 2]
```diff
@@ -16,6 +16,7 @@
 | `settings-discord-rich-presence`           | Boolean | `true` or `false`            | `false`                      | Integration with Discord Rich Presence                         |
 | `settings-start-on-boot`                   | Boolean | `true` or `false`            | `false`                      | Open player when system starts                                 |
 | `settings-start-minimized`                 | Boolean | `true` or `false`            | `false`                      | Open player minimized in tray                                  |
+| `settings-lyrics-always-top`               | Boolean | `true` or `false`            | `false`                      | Open player minimized in tray                                  |
```
Comment (mingjun97) [inline STORE_VARIABLES.md:19]: Just notice that this need to be updated as well.

File: src/utils/defaultSettings.js [hunk 3]
```diff
@@ -23,6 +23,8 @@ settingsProvider.setInitialValue('settings-miniplayer-show-task', false) // hide
 
 settingsProvider.setInitialValue('settings-lyrics-provider', '1') // OVH
 
+settingsProvider.setInitialValue('settings-miniplayer-show-top', false) // show on top always
```
Comment (mingjun97) [inline src/utils/defaultSettings.js:26]: This supposed to be `settings-miniplayer-always-top`?

Comment (flleeppyy) [reply src/utils/defaultSettings.js:26]: No actually, but looking into this there is no initial setting for `settings-miniplayer-always-top`. I guess I'll fix that

File: STORE_VARIABLES.md (hunk 2 above)
Comment (flleeppyy) [reply STORE_VARIABLES.md:19]: On it

Comment (flleeppyy) [reply STORE_VARIABLES.md:19]: Also going to sort the settings alphabetically because its really irritating me

**Pull request merged**